src/ray/gcs/format/gcs.fbs

```diff
@@ -310,3 +312,33 @@ table DriverTableData {
   // Whether it's dead.
   is_dead: bool;
 }
+
+// This table stores the actor checkpoint data. An actor checkpoint
+// is the snapshot of an actor's state in the actor registration.
+// See `actor_registration.h` for more detailed explanation of these fields.
+table ActorCheckpointData {
+  // ID of this actor.
+  actor_id: string;
+  // The dummy object ID of actor's most recently executed task.
+  execution_dependency: string;
+  // A list of IDs of this actor's handles.
+  handle_ids: [string];
+  // The task counters of the above handles.
+  task_counters: [long];
+  // The frontier dependencies of the above handles.
+  frontier_dependencies: [string];
+  // A list of unreleased dummy objects from this actor.
+  unreleased_dummy_objects: [string];
+  // The numbers of dependencies for the above unreleased dummy objects.
+  num_dummy_object_dependencies: [int];
+}
+
+// This table stores the actor-to-available-checkpoint-ids mapping.
+table ActorCheckpointIdData {
```

Comment:
I didn't do that because of efficiency. `checkpoint_ids` might be modified pretty frequently (currently we need to do checkpoint for every task to bypass replaying tasks). 
`ActorTable` is a log, if we put `checkpoint_ids` in `ActorTable`, we'll need to keep all the histories, which isn't necessary.